src/webportal/src/app/vc/vc.component.ejs

```diff
@@ -43,15 +47,30 @@
                 <td>
                   <%= data[vcName]['numJobs'] %>
                 </td>
+                <td width="8%">
+                  <a onclick="changeVcState('<%= vcName %>', '<%= data[vcName]['status'] %>')" class="state-vc state-<%= data[vcName]['status'].toLowerCase() %> <%= admin ? '' : 'state-vc-ordinary' %>">
+                    <%= data[vcName]['status'].toLowerCase().replace(/( |^)[a-z]/g, (L) => L.toUpperCase()) %>
```

Comment:
Why leading space in regular expression? Isn't status field "running" or "stopped"?

Reply:
The status responded is "RUNNING" or "STOPPED". I think it is some what inconsistent with the job's  status. So it will be changed to "Running" or "Stopped".